presentation/src/main/java/com/shifthackz/aisdv1/presentation/activity/AiStableDiffusionViewModel.kt

```diff
@@ -59,4 +64,8 @@ class AiStableDiffusionViewModel(
             homeRouter.navigateToRoute(intent.route)
         }
     }
+
+    fun hideSplash() {
+        _isShowSplash.value = false
+    }
```

Comment:
Do not expose this viewModel method. Please create another MviIntent for this specific action.

Reply:
Resolved.